README.adoc

```diff
@@ -331,9 +331,9 @@ Before you can use the filter class that you created, you need to create the `mi
 ----
```

Comment:
This `Create` block highlights the wrong file

Reply:
Line 325, hotspot the file reference

Reply:
Based on the guide's original format, creating a hotspot for the file seems excessive since the arrowhead below references the same file

Reply:
> This `Create` block highlights the wrong file

Updated.

Reply:
> This `Create` block highlights the wrong file

Going to open a separate issue on the repo since the issue is also on master, and not the only guide. (Could be a ui issue)

Reply:
Unresolved, no need to open a separate issue + PR, it would be easier to just fix it here.